doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -254,10 +254,13 @@ Matter samples
   * The :file:`prj_no_dfu.conf` file.
   * Support for ``no_dfu`` build type for nRF5350 DK, nRF52840 DK and nRF7002 DK.
 
-* Added test event triggers to all Matter samples.
-  By utilizing the test event triggers, you can simulate various operational conditions and responses in your Matter device without the need for external setup.
+* Added:
+
+   * Test event triggers to all Matter samples.
+     By utilizing the test event triggers, you can simulate various operational conditions and responses in your Matter device without the need for external setup.
 
-  To get started with using test event triggers in your Matter samples and to understand the capabilities of this feature, refer to the :ref:`ug_matter_test_event_triggers` page.
+     To get started with using test event triggers in your Matter samples and to understand the capabilities of this feature, refer to the :ref:`ug_matter_test_event_triggers` page.
+   * Support for emulation of nRF7001 Wi-Fi companion IC on nrf7002 DK to Matter Lock Sample.
```

Comment:
```suggestion

* :ref:`matter_lock_sample` sample:

   * Added support for emulation of the nRF7001 Wi-Fi companion IC on the nRF7002 DK.
```
So far, we've had changes specific to a single sample under an entry for that sample.